app/controllers/api/braintree_controller.rb

```diff
@@ -20,12 +20,12 @@ def options
     {
       nonce: params[:payment_method_nonce],
       user: params[:user],
-      amount: params[:amount].to_i
+      amount: params[:amount].to_i,
+      store: Payment
```

Comment:
Can you explain me what assigning `Payment` does here? It's not totally clear to me from looking at the Payment module.